src/App.tsx

```diff
@@ -1,9 +1,23 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
+const filtrateMovies = (movies: Movie[], query: string) => {
```

Comment:
What about this way?
```suggestion
const getMatchingMovies = (movies: Movie[], query: string) => {
```